app/templates/app/tasks.html

```diff
@@ -26,7 +49,31 @@ <h1>{% trans "Tất cả nhiệm vụ" %}</h1>
        {% trans " Users:" %} {{ task.user.all|join:", " }}
         <br>
        
-        <a href="" class="btn btn-danger">{% trans "Delete" %}</a>
+        <!-- Button trigger modal -->
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+  Delete
+</button>
+       <!-- Modal -->
+<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">{% trans "Are u sure to delete this task?" %}</h5>
```

Comment:
Nội dung text cần chuẩn chỉ e nhé, ko viết tắt nha
```suggestion
        <h5 class="modal-title" id="exampleModalLabel">{% trans "Are you sure to delete this task?" %}</h5>
```